### DISCUSSION TITLE
RADE

### DISCUSSION THREAD

**Discussion by TomF8COD** [sh123/esp32_loradv#15] [General]
Hi @sh123, I read a page about RADE, the new encoder used in Freedv, have you heard about it ? https://freedv.org/davids-freedv-update-september-2024/ From the samples provided on the page it sounds impressive, it's derived from Opus as I understand. I thought you might find it interesting if you missed it :-)

**Comment by sh123** [comment]
Hi, looks interesting, but need more time to learn all that. I tried Opus and it was giving quite good quality, but it does not support such low bit rates as Codec2.

**Reply by TomF8COD** [reply]
I want to build your devices to try opus, but I'm lacking of free time to do it. But it's on my list, with some 2.4ghz lora devices to have plenty of bandwidth to play.

**Comment by n0p** [comment]
RADAE needs a lot of horsepower, the usual ESP32 simply isn't powerful enough:
https://freedv.org/measuring-esp32-s3-performance-for-radae/

Maybe the ESP32-P4 is powerful enough, but it is not available yet.
